pyproject.toml

```diff
@@ -3,8 +3,9 @@ line-length = 100
 target-version = ['py37']
 force-exclude = '''
 /(
-  | mlflow/protos
-  | tests/protos
-  | examples/recipes
-)/
+    mlflow/protos/
```

Comment:
is the `|` not needed for the first element?

Reply:
No. `force-exclude` is a verbose regular expression. I think the right regex here is `(a|b|c)`, not `(|a|b|c)`.

Reply:
`/` at line 5 represents the root.